params/config.go

```diff
@@ -335,7 +335,8 @@ type ChainConfig struct {
 	IsDevMode bool          `json:"isDev,omitempty"`
 
 	// <specular modification>
-	EnableL2EngineApi bool `json:"enableL2EngineApi,omitempty"`
+	EnableL2EngineApi  bool           `json:"enableL2EngineApi,omitempty"`
+	L2BaseFeeRecipient common.Address `json:"l2BaseFeeRecipient,omitempty"`
```

Comment:
what are your thoughts on setting this here instead of in `params/protocol_params.go` like optimism does?
that's were the `L1FeeRecipient` is configured in my PR rn. don't really have strong opinions but feels like both should be in the same place.

Reply:
[for reference](https://github.com/ethereum-optimism/op-geth/blob/25fd986a27b206b8454b2e9ceb4b699a238974ed/params/protocol_params.go#L25)

Reply:
in the L1Fee PR, we're already configuring `EnableL1Fee` in `chainConfig`, we'd just need to change it to an address, and we can use the zero address to disable the l1 fee feature.

Reply:
that seems reasonable, i'll change `EnableL1Fee` to the address of the L1 fee recipient